recipes/numdifftools/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "numdifftools" %}
+{% set version = "0.9.20" %}
+{% set sha256 = "443b43e43c970b4f37cd2a5a57f842192da89daa2d58105ca5f5f894b7d4fcd0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-py2.py3-none-any.whl 
+  url: https://files.pythonhosted.org/packages/py2.py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py2.py3-none-any.whl 
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # I build from the wheel so that we don't need pyscaffold. The version
+  # needed by the source distribution of numdifftools is not compatible with conda-forge
```

Comment:
Why are conda-forge and pyscaffold incompatible?

Reply:
It is not really that conda-forge is incompatible with pyscaffold, instead it is numdifftools that is incompatible with the version of pyscaffold contained in conda-forge as of today. 

The version of `pyscaffold` in conda is more recent than the one that was used for numdifftools. The `setup.py` of numdifftools lists:

`pyscaffold>=2.4rc1,<2.5a0`

while the version in conda forge is 2.5.8. Pyscaffold changed significantly between these versions and the build will fail.

Numdifftools is a pure-python package, thus the wheel just contains the source code (but already preprocessed by the appropriate version of pyscaffold). I figured that in the long run it would be easier to maintain the build from the pre-processed source.

Reply:
Building from a wheel is a bit of an [anti-pattern for conda-forge](https://gitter.im/conda-forge/conda-forge.github.io?at=5a315763232e79134d45b7ea). I am checking on the policy for back dating versions into feedstocks. It may be possible to build the version you need in the pyscaffold feedstock.

Reply:
You might consider opening an issue with the pyscaffold feedstock asking for a branch with the version you need. (feel free to ping me on the issue)

Reply:
It turns out that there is already a v 2.4.2 of pyscaffold in conda-forge. This wasn't mentioned in the conversation started by somebody else on the github repository of numdifftools, but it makes things easier. I just fixed this.